jsonschema-validation.xml

```diff
@@ -491,6 +507,19 @@
                     </t>
                 </section>
 
+                <section title="requiredAllExcept">
+                    <t>
+                        The value of this keyword MUST be an array.
+                        Elements of this array, if any, MUST be strings, and MUST be unique.
+                    </t>
+                    <t>
+                        An object instance is valid against this keyword if every property name
+                        declared in "properties" within the same schema object is the name of
```

Comment:
We should consider the advantages/disadvantages of having this keyword use annotations from `properties`, which would allow it to use property definitions from other schemas via composition.

I could see this pattern being quite useful:

```
  $defs:
    model.user:
      type: object
      properties:
        id:
          type: string
        name:
          type: string
        type:
          enum: [foo, bar]
        comment:
          type: [string, 'null']
    create_operation:
      $ref: '#/$defs/model.user'
      requireAllExcept: [id, comment]
    get_operation:
      $ref: '#/$defs/model.user'
      requireAllExcept: []
```

Reply:
_[briefly de-lurks]_
@karenetheridge it's worth considering that these might be two separate use cases.  "require all of these properties right here" is a straightforward thing that would become hard if this always used annotations from applicators in subschemas.  And trying to make it optional within the keyword would introduce more complex syntax.
_[re-lurks]_